<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#e0cfb1;
color:#e0cfb1;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="clear_all">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #a5c3ec;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>отчистить все задачи</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_task">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #a5c3ec;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>добавить задачу</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="task_name">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
background-color: #a5c3ec;
border-radius:20px;
}</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>201</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
